modules/claims_api/spec/requests/v2/veterans/rswag_disability_compensation_request_spec.rb

```diff
@@ -188,20 +191,139 @@
+                                   'disability_compensation', 'form_526_json_api.json').read
+            temp = JSON.parse(temp)
+
+            temp
+          end
+
+          let(:scopes) { %w[system/claim.write] }
+          let(:auto_claim) { create(:auto_established_claim) }
+          let(:attachment1) do
+            Rack::Test::UploadedFile.new(Rails.root.join(*'/modules/claims_api/spec/fixtures/extras.pdf'.split('/'))
+                                                     .to_s)
+          end
+          let(:attachment2) do
+            Rack::Test::UploadedFile.new(Rails.root.join(*'/modules/claims_api/spec/fixtures/extras.pdf'.split('/'))
+                                                     .to_s)
+          end
+          let(:id) { auto_claim.id }
+
+          before do |example|
+            stub_poa_verification
+            stub_mpi
+
+            mock_ccg(scopes) do
+              VCR.use_cassette('claims_api/disability_comp') do
+                submit_request(example.metadata)
+              end
+            end
+          end
+
+          after do |example|
+            example.metadata[:response][:content] = {
+              'application/json' => {
+                example: JSON.parse(response.body, symbolize_names: true)
+              }
+            }
+          end
+
+          it 'returns a valid 200 response' do |example|
+            assert_response_matches_metadata(example.metadata)
           end
         end
       end
 
       describe 'Getting a 401 response' do
         response '401', 'Unauthorized' do
-          it 'returns a 401 response' do
-            one = 1
-            expect(one).to eq(1)
+          schema JSON.parse(Rails.root.join('spec', 'support', 'schemas', 'claims_api', 'v2', 'errors',
+                                            'default.json').read)
+
+          let(:data) do
+            temp = Rails.root.join('modules', 'claims_api', 'spec', 'fixtures', 'v2', 'veterans',
+                                   'disability_compensation', 'form_526_json_api.json').read
+            temp = JSON.parse(temp)
+
+            temp
+          end
+
+          let(:scopes) { %w[system/claim.write] }
+          let(:auto_claim) { create(:auto_established_claim) }
+          let(:attachment1) do
+            Rack::Test::UploadedFile.new(Rails.root.join(*'/modules/claims_api/spec/fixtures/extras.pdf'.split('/'))
+                                                     .to_s)
+          end
+          let(:attachment2) do
+            Rack::Test::UploadedFile.new(Rails.root.join(*'/modules/claims_api/spec/fixtures/extras.pdf'.split('/'))
+                                                     .to_s)
+          end
+          let(:id) { auto_claim.id }
+          let(:Authorization) { nil }
+
+          before do |example|
+            stub_poa_verification
+            stub_mpi
+
+            VCR.use_cassette('claims_api/disability_comp') do
```

Comment:
also, not a big deal, we can move this `use_cassette` up to the top describe rather than stating it in every test.